Point2D position
float32 yaw
float64[36] covariance
